specs/commandline/sign.md

```diff
@@ -153,6 +165,30 @@ Warning: Always sign the artifact using digest(`@sha256:...`) rather than a tag(
 Successfully signed localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
 ```
 
+### Sign an arbitrary file located in file system
+Notation supports signing a file located in user's file system. The file's content (blob) gets signed and the generated signature is stored in the same file system.
```

Comment:
```suggestion
Notation supports signing files located in file systems.
```

Reply:
File systems can be local (e.g. `ntfs`, `ext4`, `apfs`) or remote (`cifs`) or anything. Since the signed content and the generated signatures are files, they can be on any file systems and can be on different file systems.